app/views/profiles/search_results.html.erb

```diff
@@ -1,5 +1,38 @@
-<% @profile_results.each do |profile_result| %>
-  <h3><%= profile_result.username %></h3>
-<% end %>
+<div class="row">
+	<div class="col-sm-3">
+		<%= form_tag('/profiles/search_results', method: :get) %>
+      <%= label_tag(:filter_gender, "Gender") %>
+      <%= select_tag("search[gender]", options_for_select(
```

Comment:
Might be a good use of a helper to move the `options_for_select` call out of the view.